squarelet/organizations/serializers.py

```diff
@@ -161,6 +164,9 @@ class Meta:
             "avatar": {"required": False},
         }
 
+    def get_plan(self, obj):
+        return obj.plan.slug if obj.plan else "free"
```

Comment:
The plan property on `Organization` is a shim which gets the MuckRock plan for the organization, as my code used to assume an Organization can only have one plan - now PressPass model has multiple plans, so adding this like this makes no sense.  Plans are linked to organizations through subscriptions, which are viewable as a subresource - `/pp-api/organizations/{uuid}/subscriptions/`